autoswap/lightning.go

```diff
@@ -337,15 +351,21 @@ func (cfg *LightningConfig) walletId() *database.Id {
 	return nil
 }
 
-func (cfg *LightningConfig) execute(recommendation *LightningRecommendation) error {
+func (cfg *LightningConfig) Execute(recommendation *autoswaprpc.LightningRecommendation) error {
+	if len(recommendation.Swap.DismissedReasons) > 0 {
+		logger.Infof("Skipping swap recommendation %+v", recommendation.Swap)
```

Comment:
Is there a scenario in which you would still want to manually execute a recommendation despite it being dismissed? Like eg I need inbound *right now* and am willing to pay over budget?